Makefile

```diff
@@ -23,6 +23,9 @@ $(module)/talk.html: $(module)/talk.rst
 
 install: $(BIN)
 	install $(BIN) $(PREFIX)/bin
+	@mkdir -p /etc/bash_completion.d
```

Comment:
Does it really make sense to create this folder? If it’s not already there, then the system probably stores its bash completion scripts elsewhere, and this folder probably won’t be sourced.